bin/jest.js

```diff
@@ -223,6 +223,17 @@ if (fs.existsSync(cwdJestBinPath)) {
   }
 }
 
+(function printLegacyNodeDeprecationMessage() {
+  if (!/^v0/.test(process.version)) {
+    return;
+  }
+
+  console.log('\n  == NOTICE: ==');
+  console.log('    On August 1st, 2015, Jest v0.5.x will work only on io.js');
+  console.log('    Legacy v0.4.x for Node 0.10.x can be found in the ' +
+    'appropriate branch.\n');
```

Comment:
It would be helpful to provide more specific instructions on how to use 0.4.x. Just an idea: "Legacy v0.4.x for Node 0.10.x will still be available on npm and from the "0.4.x" branch on GitHub."